siac/main.go

```diff
@@ -29,8 +28,10 @@ func apiGet(call string) (*http.Response, error) {
 	if host, port, _ := net.SplitHostPort(addr); host == "" {
 		addr = net.JoinHostPort("localhost", port)
 	}
-
-	resp, err := http.Get("http://" + addr + call)
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "http://"+addr+call, nil)
```

Comment:
use `req, _ :=` to show your intent; otherwise it looks like the error is just being overwritten by the subsequent call to `client.Do`.